code/src/App.js

```diff
@@ -1,12 +1,24 @@
 import React from 'react'
 import data from './data.json'
+import { Album } from './components/Album'
```

Comment:
You don't need the brackets when importing only one thing :)
```suggestion
import Album from './components/Album'
```